bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -2312,6 +1979,7 @@ public void setBackground(Color color) {
 
 	/* Get the HDC for the device */
 	long hDC = device.internal_new_GC(null);
+	long handle = win32_getHandle(this, getZoom());
```

Comment:
Hmm, why using this handle? Shouldn't that be applied to all handles?

Reply:
Hmm. Earlier, it was just done by the handle. But you are right. It means it was a wrong implementation already.

Reply:
I assume your comment has been addressed, @akoch-yatta ?

The current implementation calls `setBackground(Color color, long handle)` for each handle in `zoomLevelToImageHandle`:

```java
public void setBackground(Color color) {
	...
	zoomLevelToImageHandle.values().forEach(imageHandle -> setBackground(color, imageHandle.handle));
}

private void setBackground(Color color, long handle) {
	...
	/* Get the HDC for the device */
	long hDC = device.internal_new_GC(null);

	/* Change the background color in the image */
	BITMAP bm = new BITMAP();
	OS.GetObject(handle, BITMAP.sizeof, bm);
	long hdcMem = OS.CreateCompatibleDC(hDC);
	OS.SelectObject(hdcMem, handle);
	...
}
```